Create Updated main.py

shaypheraddison opened the pull request:
Is this the kind of idea for creating a function? Or can it be simplified even further?

shaypheraddison committed:
Create Updated main.py

resoluteCoder reviewed (commented):
I think this a great start and exactly what I was referring to. 

Let's do the same with accepting the input from the user that also returns the conversion in the proper type. Then store the two in separate variables which then get passed to the calc function

shaypheraddison committed:
Update Updated main.py

I hope this isn't too excessive

shaypheraddison reviewed (commented):
Is this what you were saying? I feel like its a little excessive and don't know how to consolidate this much

shaypheraddison committed:
Update Updated main.py

shaypheraddison committed:
Update Updated main.py

shaypheraddison committed:
Update Updated main.py

resoluteCoder reviewed (commented):
I like the addition of the list. Now you should be able to loop over each item and use the item in the print state of the input. This way you have one input statement but this will also require having to change how you declared/init your variables.

What you could do is look into dictionaries and use a single dict for all the values.

shaypheraddison committed:
Update Updated main.py

shaypheraddison committed:
Merge branch 'Updates' of https://github.com/shaypheraddison/Baseball-Stat-Line-Calculator into Updates

shaypheraddison committed:
Update Updated main.py

Cant seem to figure out how to call the input results for the functions, still looking into that. However, the list and the loop are working as I'd like it to

shaypheraddison committed:
Update Updated main.py

shaypheraddison commented:
I think I broke it. I had it working through the loop at one point today. it was even converting it to a float number, but now I keep getting an error when I have either float or int in front of the input. the error shows up after entering the first number too.

shaypheraddison committed:
Update Updated main.py

shaypheraddison committed:
Update Updated main.py

I....did it

resoluteCoder commented on Updated main.py:
```diff
@@ -0,0 +1,59 @@
+def get_user_input():
```
this code can be condensed to:
```python
stats = {}
labels = ["hits", "at-bats"] #the rest ...
for label in labels:
    str_value = input(f'Enter the number of {label}')
    stats[label] = int(str_value)
return stats
```

resoluteCoder commented on Updated main.py:
```diff
@@ -0,0 +1,59 @@
+    slg = calculate_slugging_percentage(singles, doubles, triples, home_runs, at_bats)
+    return obp + slg
+
+def calculate_and_display_slash_line(stats):
```
Let's change this from doing two things. Remember you want functions to do one thing (if you can help it)

try return the calculations in a format that can be sent do a `display_slash_line` function

resoluteCoder reviewed (changes requested):
Very nicely done a couple of improvements but otherwise very impressed! Let's keep it up!!

shaypheraddison committed:
Update Updated main.py
